README.md

```diff
@@ -100,6 +100,7 @@ All parameters are optional.
 | `noConflict` | `true` | By default, this plugin will read from the `rtemplate` key in your `yaml` front matter. However, if this is the only templating plugin, you may set `noConflict` to `false` to use the `template` key instead.
 | `pattern` | `**/*` | Specifies a file filter pattern.
 | `preserve` | `false` | Stores a copy of un-templated contents into `rawContents` meta which you can access in your React components.
+| `preserveProps` | `false` | Stores a copy of the props objects provided to the Reach template into `props` which you can access in downstream plugins.
```

Comment:
`Reach` -> `React` 

`props objects` -> `props`

`into props` -> `into the props meta`